driver/controller.go

```diff
@@ -29,9 +29,14 @@ type controllerService struct {
 func (s controllerService) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest) (*csi.CreateVolumeResponse, error) {
 	capabilities := req.GetVolumeCapabilities()
 	source := req.GetVolumeContentSource()
+	deviceClass, ok := req.Parameters[topolvm.DeviceClassKey]
+	if !ok {
+		deviceClass = topolvm.DefaultDeviceClassName
+	}
```

Comment:
As the default device class is empty, this can be as simple as
```suggestion
	deviceClass := req.Parameters[topolvm.DeviceClassKey]
```